lib/workos/webhooks.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+# typed: true
+
+require 'json'
+require 'openssl'
+
+module WorkOS
+  # The Webhooks module provides convenience methods for working with the WorkOS webhooks.
+  # You'll need to extract the signature header and payload from the webhook request
+  # sig_header = request.headers['WorkOS-Signature']
+  # payload = request.body.read
+  #
+  # The secret is the Webhook Secret from your WorkOS Dashboard
+  # The tolerance is for the timestamp validation
+  #
+  module Webhooks
+    class << self
+      extend T::Sig
+      include Base
```

Comment:
Do we need to include `Base` here? Seems like we don't use any of its behavior, and the specs still pass without it.